R/fit_model.R

```diff
@@ -0,0 +1,52 @@
+fit_model <- function(data, formula, ...) {
+  UseMethod("fit_model")
+}
+
+#' @export
+fit_model.RtGam_gam <- function(
+    data,
+    formula,
+    family = "nb",
+    method = "REML",
+    ...) {
+  # Override the defaults in formals with the user-supplied args in dots
+  formal_arg_names <- Filter(function(x) x != "...", names(formals()))
```

Comment:
if my other suggestion doesn't work this might need to be a internal function to avoid duplication?

Reply:
Resolved by previous.